### DISCUSSION TITLE
Dpkg broken after upgrade 6.1 Jolnir

### DISCUSSION THREAD

**Discussion by devlinarptiw** [elementary/os#640] [Q&A] [answered]
Ran apt upgrade and things got stuck at some dpkg issue and now cant install anything since it keeps getting stuck at same issue. Following is an example from trying to install some git client

_Setting up ca-certificates (20211016~20.04.1) ...
Updating certificates in /etc/ssl/certs...
sed: read error on /usr/share/ca-certificates/mozilla/Microsoft_RSA_Root_Certificate_Authority_2017.crt: Input/output error
dpkg: error processing package ca-certificates (--configure):
installed ca-certificates package post-installation script subprocess returned error exit status 4
Errors were encountered while processing:
ca-certificates
E: Sub-process /usr/bin/dpkg returned an error code (1)_

**Comment by ryonakano** [comment, marked as answer]
It says `Input/output error` so there would be some errors in the hardware layer of your rootfs, e.g. cable connection is loose or the storage is going to die.

Sorry for late reply, closing as (probably) resolved.
